src/extensions/extension-manager.ts

```diff
@@ -8,9 +8,10 @@ import { extensionPackagesRoot } from "./extension-loader"
 import { getBundledExtensions } from "../common/utils/app-version"
 
 export interface InstalledExtension {
```

Comment:
shouldn't this be a class? That way there isn't an abstraction boundary being broken by the loader.